### PR TITLE
Room Disconnect Message

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
Add Message when disconnect instead of reconnecting
screen shot 2018-07-02 at 3 39 42 pm

### CODE REVIEW

File: view/rooms/app.js [hunk 1]
```diff
@@ -754,6 +754,26 @@ function setupSocketIOClient(bots) {
 
   socket.on('disconnect', () => {
     debugMessage('Socket Disconnected');
+    const elem = document.createElement('div');
+    elem.id = 'snackbar';
+    elem.className = 'slds-notify slds-notify_toast slds-theme_offline show';
+    elem.innerHTML = '<div class="snackContent">You have ' +
+      'lost connection with Refocus, so bot data and bot actions ' +
+      'maybe unreliable. Please save any unfinished work and ' +
```
Comment (iamigo) [inline view/rooms/app.js]: `maybe` should be `may be`

File: view/rooms/app.js [hunk 2]
```diff
@@ -754,6 +754,26 @@ function setupSocketIOClient(bots) {
...
+      '<a href="javascript:window.location.reload(true)">' +
+      'click here to reconnect</a></div>';
```
Comment (iamigo) [inline view/rooms/app.js]: the link should just be the word `here` (not the whole phrase) OR get rid of the word "here" and just say "... and click reconnect to continue." where "reconnect" is the link.

File: public/css/room.css [hunk 3]
```diff
@@ -45,3 +45,32 @@
     opacity: 0;
   }
 }
+
```
Comment (kriscfoster) [inline public/css/room.css]: The indentation seems to be a bit off in some of these

**Comment by tausifmuzaffar** [comment]
@kriscfoster @iamigo updated

File: view/rooms/app.js [hunk 4]
```diff
@@ -754,6 +754,26 @@ function setupSocketIOClient(bots) {
 
   socket.on('disconnect', () => {
     debugMessage('Socket Disconnected');
+    const elem = document.createElement('div');
+    elem.id = 'snackbar';
+    elem.className = 'slds-notify slds-notify_toast slds-theme_offline show';
+    elem.innerHTML = '<div class="snackContent">You have ' +
+      'lost connection with Refocus, so bot data and bot actions ' +
```
Comment (iamigo) [inline view/rooms/app.js]: no comma between "Refocus" and "so"

**Pull request merged**